cmd/nerdctl/compose_config_test.go

```diff
@@ -75,3 +76,31 @@ services:
 	newHash := base.ComposeCmd("-f", newComp.YAMLFullPath(), "config", "--hash=hello1").Out()
 	assert.Assert(t, hash != newHash)
 }
+
+func TestComposeConfigWithMultipleFile(t *testing.T) {
+	base := testutil.NewBase(t)
+
+	var dockerComposeYAML = `
+services:
+  hello1:
+    image: alpine:3.13
```

Comment:
plz use  testing predefined image https://github.com/containerd/nerdctl/blob/251ddcef7bac29f53241ed36d38038ed852089bb/pkg/testutil/testutil_linux.go#L36

Reply:
@fahedouch Only the config is tested here, and the image will not be pulled. It may be better to explicitly write the image version.